app/views/layouts/my_facilities.html.erb

```diff
@@ -79,7 +84,7 @@
             </div>
           </div>
           <div class="modal-footer">
-            <button type="button" class="btn btn-secondary" data-dismiss="modal">Clear all</button>
+            <a href="?" type="button" class="btn btn-secondary">Clear all</a>
```

Comment:
Why a `?` href? Can we use `#` instead?

Reply:
The `?` clears the query params, and therefore clears out the filters. The `#` doesn't do that.

Reply:
Oooooh 👍